### DISCUSSION TITLE
66BFE39D-47F4-4F56-923C-437976473B82

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#208] [General]
# Two Comets Might Become the Brightest Seen from Earth since 1965!

Week 41, 2024: Two Comets Might Become the Brightest Seen from Earth since 1965!

https://celestia.mobi/article?guide=66BFE39D-47F4-4F56-923C-437976473B82&lang=es

**Comment by Joelsanjur** [comment]
Super

**Comment by chant-19OX** [comment]
Cool

**Comment by Candy370** [comment]
Spectacular

**Comment by kmjordan122313-lgtm** [comment]
comets of 2024 and 2025, spectacular.
